app/src/main/java/com/keylesspalace/tusky/StatusListActivity.kt

```diff
@@ -178,8 +185,9 @@ class StatusListActivity : BottomSheetActivity(), HasAndroidInjector {
             mastodonApi.getFilters().fold(
                 { filters ->
                     mutedFilter = filters.firstOrNull { filter ->
+                        // TODO shouldn't this be an exact match (only one keyword; exactly the hashtag)?
```

Comment:
That's probably worth checking.  This was originally used with v1 filters, where you could only specify one keyword (or phrase) per filter.

Reply:
Although you need to be careful here, since always adding the "#" will mean that hashtags that were previously shown as muted will no longer be shown as muted, possibly confusing users. It might be worth checking both.

Reply:
Maybe couple it with checking the title of the filter?

Reply:
Hm, as that filter can be changed freely afterwards (i.e. adding other keywords) this is always a heuristic, I guess.

With tapping "unmute" one could destroy (involuntarily) some config work.

Or conversely with an exact match, the state of the button could be wrong; and you add and remove a doubled filter.

Reply:
Sounds reasonable (and a bit involved).

I'd do it in another issue or at least another PR - as the problem is not new here.